src/GiphyClient.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Eve\Command;
+
+use GuzzleHttp\Client;
+
+class GiphyClient
+{
+    private $client;
+
+    public function __construct()
+    {
+    }
+
+    public function getImageFor(string $search, string $key)
+    {
+        $this->client = new Client([
+            'base_uri' => 'http://api.giphy.com/v1/gifs/'

```

Comment:
This would be better as a configuration option or even a class constant, instead of hard-coded.